src/components/repl.tsx

```diff
@@ -82,27 +82,65 @@ const Repl: ReplProps = (props) => {
   }
   const [edit, setEdit] = createSignal(-1);
   const [outputTab, setOutputTab] = createSignal(0);
+  let idx = props.tabs.findIndex((tab) => tab.name === 'data_import_map.json');
+  let import_map = {};
+  if (idx >= 0) {
+    import_map = JSON.parse(unwrap(props.tabs[idx]).source);
+  }
+  const [importMap, setImportMap] = createSignal<any>(import_map, { equals: false });
```

Comment:
maybe we could create a `type ImportMap = Record<string, string>` or something?

Reply:
Just implemented this